src/main/java/ch/njol/skript/entity/EntityData.java

```diff
@@ -462,18 +462,13 @@ public E[] getAll(final World... worlds) {
 	public static <E extends Entity> E[] getAll(final EntityData<?>[] types, final Class<E> type, @Nullable World[] worlds) {
 		assert types.length > 0;
 		if (type == Player.class) {
-			if (worlds == null && types.length == 1 && types[0] instanceof PlayerData && ((PlayerData) types[0]).op == 0)
+			if (worlds == null)
 				return (E[]) Bukkit.getOnlinePlayers().toArray(new Player[0]);
-			final List<Player> list = new ArrayList<>();
-			for (final Player p : Bukkit.getOnlinePlayers()) {
-				if (worlds != null && !CollectionUtils.contains(worlds, p.getWorld()))
+			List<Player> list = new ArrayList<>();
+			for (Player p : Bukkit.getOnlinePlayers()) {
+				if (!CollectionUtils.contains(worlds, p.getWorld()))
 					continue;
-				for (final EntityData<?> t : types) {
-					if (t.isInstance(p)) {
 						list.add(p);
```

Comment:
I think this line is supposed to be indented less?
![image](https://user-images.githubusercontent.com/29044720/187094635-4d286164-740d-406f-9e5b-4f394767e847.png)